content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst

```diff
@@ -0,0 +1,139 @@
+=================
+Calls for Tenders
+=================
+
+**Calls for Tenders** are one of the two main types of **Purchase Agreements** available by default
+in Odoo.
+
+*Calls for Tenders* are used to request offers from multiple vendors at the same time, by inviting
+those vendors to submit offers for similar goods or services all at once. This helps companies that
+use this feature to select the cheapest and fastest vendors in the market, depending on their
+needs.
+
```

Comment:
...use this feature to select the cheapest (and fastest) vendors, depending on their specific business needs.